dijet/selection/lepton_selection.py

```diff
@@ -24,16 +24,17 @@ def lepton_selection(
     # lepton selection based on old UHH2 framework
     # https://github.com/UHH2/DiJetJERC/blob/ff98eebbd44931beb016c36327ab174fdf11a83f/src/AnalysisModule_DiJetTrg.cxx#L703
     # IDs in JME Nano https://cms-nanoaod-integration.web.cern.ch/integration/master-106X/mc102X_doc.html
-    # mask for muons
+    events = attach_coffea_behavior(events, collections={"Muon": { "type_name": "Muon","skip_fields": "*Idx*G",}})
```

Comment:
I'm a bit puzzled why this is needed. We don't use any behaviors here for the leptons, do we? Plus this seems to add the behaviors only for the muon collection. Are there issues if running without attaching the coffea behavior explicitly?